Doc/library/dbm.rst

```diff
@@ -8,7 +8,16 @@
 
 --------------
 
-:mod:`dbm` is a generic interface to variants of the DBM database:
+A DBM database is a persistent, dictionary-like object that stores keys and values as :class:`bytes`.
```

Comment:
A DBM database is not a dictionary-like object. The `dbm` package provides a dictionary-like interface to the DBM databases.

Reply:
You are right. How would you define a DBM database? It's hard for me to explain what it is for someone who hasn't used a dbm library.

Reply:
```suggestion
The :mod:`!dbm` module is a generic interface to key-value databases,
that stores keys and values as :class:`bytes`.
```

Reply:
I'm not sure that any changes are needed here, so I left this on others core developers which can be more interesting in changes.

Reply:
I agree that this particular change is not needed. However, I think it may make sense to make the "seealso" more prominent.

Reply:
Alright, not touching this paragraph.